hrms/svc/service.py

```diff
@@ -0,0 +1,60 @@
+from hrms.prst import model
+from sqlmodel import Session, select
+
+class Employee:
+    def list(self):
+        with Session(model.engine) as session:
+            statement = select(model.Employee)
+            results = session.exec(statement)
+            employees = results.all()
+            return employees
```

Comment:
Every time we use a session inside the method, the method is in control of the session. What if I want to inject the session or control the session from outside?